scripts/test.sh

```diff
@@ -85,6 +85,9 @@ else
                 cross=true
                 shift
                 ;;
+            *)
+                shift
+                ;;
```

Comment:
This would be triggered when one passes an argument which isn't one of the supported ones, right ?
In this case, I think that we want to print and error message followed by "exit 1".